lib/pushmi_pullyu/fedora_object_fetcher.rb

```diff
@@ -0,0 +1,59 @@
+require 'pushmi_pullyu'
+require 'net/http'
+require 'fileutils'
+
+class PushmiPullyu::FedoraObjectFetcher
+
+  attr_accessor :config, :noid
+
+  RDF_FORMAT = 'text/rdf+n3'.freeze
+
+  def initialize(noid, config = nil)
+    self.noid = noid
+    self.config = config || PushmiPullyu.options
```

Comment:
I would keep the same name of "options" too instead of using "config"? 

It's better for a CTRL-F and seeing whos using stuff....for example if i wanted to refactor `options[:fedora][:url]` to say....`optons[:fedora][:host]:.optons[:fedora][:port]` etc. Much harder if there named differently etc.